pkg/render/logstorage/linseed/linseed_test.go

```diff
@@ -86,9 +86,10 @@ var _ = Describe("Linseed rendering tests", func() {
 				PullSecrets: []*corev1.Secret{
 					{ObjectMeta: metav1.ObjectMeta{Name: "tigera-pull-secret"}},
 				},
-				KeyPair:       kp,
-				TrustedBundle: bundle,
-				ClusterDomain: clusterDomain,
+				KeyPair:         kp,
+				TrustedBundle:   bundle,
+				ClusterDomain:   clusterDomain,
+				EsAdminUserName: "elastic",
```

Comment:
I think long term we're going to want separate user names for each instance of Linseed we install, but this is OK for now